doc/api/util.md

```diff
@@ -620,8 +620,35 @@ util.inspect(obj);
 added: v6.6.0
 -->
 
-* {symbol} that can be used to declare custom inspect functions, see
-[Custom inspection functions on Objects][].
+* {symbol} that can be used to declare custom inspect functions.
+
+This is a shared symbol, `Symbol.for('util.inspect.custom')`, which is
+registered in the [global symbol registry][]. It can be used to define
+custom inspection functions in any environment.
+
+```js
+const inspect = Symbol.for('util.inspect.custom');
+
+class Password {
+  constructor(value) {
+    this.value = value || document.getElementById('password').value;
```

Comment:
Not really a fan of non-Node.js examples in the Node.js documentation ;)

Reply:
[Done](https://github.com/nodejs/node/pull/20857/commits/f45f51f1666318a4f584d78f3de86fb0157f6d36).